test/module/irohad/multi_sig_transactions/transport_test.cpp

```diff
@@ -77,16 +133,6 @@ static bool statesEqual(const iroha::MstState &a, const iroha::MstState &b) {
  * @then Assume that received state same as sent
  */
 TEST_F(TransportTest, SendAndReceive) {
-  auto interface_tx_validator =
-      std::make_unique<shared_model::validation::MockValidator<
-          shared_model::interface::Transaction>>();
-  auto proto_tx_validator = std::make_unique<
-      shared_model::validation::MockValidator<iroha::protocol::Transaction>>();
-  auto tx_factory = std::make_shared<shared_model::proto::ProtoTransportFactory<
-      shared_model::interface::Transaction,
-      shared_model::proto::Transaction>>(std::move(interface_tx_validator),
-                                         std::move(proto_tx_validator));
-
   ON_CALL(*tx_presence_cache_,
```

Comment:
Not your code, though this will show a warning, so better change it to
```suggestion
   EXPECT_CALL(*tx_presence_cache_,
```
and `WillByDefault` to `WillRepeatedly`

Reply:
Changed.